frontend/packages/kubevirt-plugin/src/models/kvReferenceForModel.ts

```diff
@@ -0,0 +1,37 @@
+import * as _ from 'lodash';
+import { modelFor, referenceForGroupVersionKind } from '@console/internal/module/k8s';
+import { K8sKind } from '@console/internal/module/k8s/types';
+
+const VERSIONS = ['v1', 'v1alpha3', 'v1beta1', 'v1alpha1'];
+
+export const kvReferenceForModel = (model: K8sKind): string => {
```

Comment:
Nit: I'd suggest naming this `kubevirtReferenceForModel` but I'm also OK if you want to keep it this way.